packages/chord-symbol/src/typedefs.js

```diff
@@ -1,13 +1,26 @@
 /**
- * A data object representing a chord. It is the result of the parsing operation and can be used for rendering.
+ * An object that may be a chord data object or a chord parsing failure object. The `success` property indicates which.
+ * @typedef {Chord | ChordParseFailure} MaybeChord
```

Comment:
I'm not sure this is JSdoc compliant? https://jsdoc.app/tags-typedef.html
I'd like to keep everything typescript-specific in the type declaration file, even if that means more work to maintain it. It should not be an issue if the typescript types are slightly different than the jsdoc ones?

Reply:
Hm, yes. It seems JSDoc requires unions to be enclosed in parentheses. Unless you were thinking of more than that?

Reply:
I've added the parentheses to this and removed the `success` property. Let me know of anything else!